app/models/sent_scheduled_message.rb

```diff
@@ -0,0 +1,11 @@
+class SentScheduledMessage < ActiveRecord::Base
+  belongs_to :employee
+  belongs_to :scheduled_message
+
+  validates :employee, presence: true, uniqueness: { scope: :scheduled_message }
+  validates :message_body, presence: true
```

Comment:
The thinking behind this is that we are "de normalizing" the message body so that if it changes after it was sent, we still know what was sent. For FOIA reasons. :ghost: 

Not sure I think we *also* need message title, since that isn't sent to the employee. Thoughts?